snippets/product-thumbnail.liquid

```diff
@@ -22,17 +22,23 @@
     <span class="product__media-icon motion-reduce">{% render 'icon-play' %}</span>
```

Comment:
In the comments at the start of this file, you can add the new `media_width` "property" that it can accept :)

Reply:
👍